### ISSUE TITLE
Out of Google Play

### ISSUE THREAD

**Issue by bjesus** [archrival/Subsonic-Android#8]
Hi,
I noticed the app is out of Google Play for a while :-( 

Is it okay that I rebuild it and submit it? a lot of my friends use it daily and people are afraid of changing phones because of the app not existing anymore.

Under what license is the app released? Can I make changes to it?

Thank you for creating the best SubSonic client.

**Comment by trumad** [comment]
Oh PLEASE build an apk and link to it here! Even if it's not on google play, a genuine APK would be much appreciated. And why was it removed anyway?

**Comment by keckelt** [comment]
You might want to check out DSub. I switched to it a couple of months ago. Also has great support.
https://github.com/daneren2005/Subsonic

**Comment by trumad** [comment]
Yeah, it does seem good. I was so used to Ultrasonic, but maybe I'll switch over. The Subsonic app on the market is balls.

**Comment by bjesus** [comment]
@trumad this is what I have http://www.droidbin.com/p19p6t6chd1ohs10fk1qsu7djnc43

**Comment by addisflava** [comment]
@bjesus  Would anybody be interested in modifying this app for me for couple of bucks . I would like a home ui resign and couple of things twicked
